spec/GraphQLOverHTTP.md

```diff
@@ -215,3 +215,31 @@ as long as the server is still able to produce a well-formed response.
 
 In case of errors that completely prevent the successful execution of the request,
 the server SHOULD respond with the appropriate status code depending on the concrete error condition.
+
+The following examples provide guidance on how to deal with specific error cases:
+
+### Unparseable or invalid request body
+
+For example: `NONSENSE`, `{"qeury": "{__typena`
+
+Completely prevents execution of the GraphQL operation and SHOULD result in status code `4ß0` (Bad Request).
+
+### Document validation
+
+Includes validation steps that run before execution of the GraphQL operation:
+- [GraphQL specification validation](http://spec.graphql.org/June2018/#sec-Validation)
+- custom validation, for example: depth limit, complexity limit
+
+The server MAY choose to:
+- deny execution, the status code SHOULD be `400` (Bad Request)
+- attempt execution, the status code will depend on the validity of the result
```

Comment:
```suggestion
The server SHOULD deny execution with a status code of `400` (Bad Request).  An invalid request is still technically executable, therefore the server MAY attempt execution.  In this event, the status code will vary based on other parts of this document.
```

Reply:
Thanks, incorporated parts of this into an updated version.